update/ansible/playbook/tasks/update.yml

```diff
@@ -22,6 +22,38 @@
       - vmproxy
       - grafana-db-migrator
   pre_tasks:
+      - name: Check if ansible-core RPM was updated
+      shell: "grep -A 1 'Install ansible-core RPM          | EL9' /srv/logs/pmm-update-perform.log | grep -qE 'changed' && echo 'Updated' || echo 'Not Updated'"
```

Comment:
```suggestion
      shell: "grep -A 1 'Install ansible-core RPM          | EL9' /srv/logs/pmm-update-perform.log | grep -q 'changed' && echo 'Updated' || echo 'Not Updated'"
```

Reply:
I think we need to reverse the quotes, i.e. the outermost should be single quotes.